marketplace-builder/src/builder.rs

```diff
@@ -181,6 +181,7 @@ impl BuilderConfig {
         let proxy_global_state = ProxyGlobalState::new(
             global_state.clone(),
             (builder_key_pair.fee_account(), builder_key_pair.clone()),
+            Duration::from_secs(5),
```

Comment:
@QuentinI What should I set this to?

Reply:
That's where `max_api_timeout_duration` should go